modules/broker-amqp/src/main/java/org/wso2/broker/amqp/codec/frames/ChannelFlow.java

```diff
@@ -56,6 +57,7 @@ protected void writeMethod(ByteBuf buf) {
     public void handle(ChannelHandlerContext ctx, AmqpConnectionHandler connectionHandler) {
         AmqpChannel channel = connectionHandler.getChannel(getChannel());
         channel.setFlow(active);
+        LOGGER.debug("Channel.flow method received. ChannelId: {} active: {} ", getChannel(), active);
```

Comment:
Shouldnt we check if debug is enabled ?

Reply:
Since we are not doing any additional string manipulation and just parsing arguments with `{}` avoided the isDebugEnabled check.